chapters/8-annotations.md

```diff
@@ -144,6 +192,38 @@ For RDF, the annotations are a property of the SPDX Document, Package, File, or
 </File>
 ```
 
+**8.4.7** YAML:
+
+For YAML, the annotations are a property of the SPDX element it is annotating.
+
+```text
+Document:
+    annotations:
+        - annotationDate: "2010-01-29T18:30:22Z"
+          annotationType: "OTHER"
+          annotator: "Person: Jane Doe()"
```

Comment:
I _think_ that there is usually a space between the end of the annotator's name and the parentheses.